admin/javascript/src/state/config/actions.js

```diff
@@ -0,0 +1,14 @@
+import ACTION_TYPES from './action-types';
+
+/**
+ * Sets global config for the application.
+ *
+ * @param object config
+ */
+export function setConfig(config) {
+  return (dispatch) =>
+    dispatch({
+      type: ACTION_TYPES.SET_CONFIG,
+      payload: { config },
+    });
+}
```

Comment:
Good catch, I've fixed that both for the `config` and `schema` actions - got a bit carried away with the async pattern there hehe.